.travis.yml

```diff
@@ -65,7 +65,7 @@ script:
     #- mkdir -p coverage
     #- chmod 777 coverage
     #- cp tools/coverage.php demos/coverage.php
-    - vendor/bin/behat --config behat-travis.yml || exit -1
+    - vendor/bin/behat --config tests/behat-travis.yml || exit -1
```

Comment:
tests/behat/travis.yml?

Reply:
I do not like too much files in root directory, but I think it should not go to tests/behat, it is config...